language/de.json

```diff
@@ -0,0 +1,372 @@
+	"Artstyle Abstract": "Kunststil Abstrakt",
+	"Artstyle Abstract Expressionism": "Kunststil Abstrakter Expressionismus",
+	"Artstyle Art Deco": "Kunststil Art Déco",
+	"Artstyle Art Nouveau": "Kunststil Jugendstil",
+	"Artstyle Constructivist": "Kunststil Konstruktivismus",
+	"Artstyle Cubist": "Kunststil Kubismus",
+	"Artstyle Expressionist": "Kunststil Expressionismus",
+	"Artstyle Graffiti": "Kunststil Graffiti",
+	"Artstyle Hyperrealism": "Kunststil Hyperrealismus",
+	"Artstyle Impressionist": "Kunststil Impressionismus",
+	"Artstyle Pointillism": "Kunststil Pointillismus",
+	"Artstyle Pop Art": "Kunststil Pop-Art",
+	"Artstyle Psychedelic": "Kunststil Psychedelisch",
+	"Artstyle Renaissance": "Kunststil Renaissance",
+	"Artstyle Steampunk": "Kunststil Steampunk",
+	"Artstyle Surrealist": "Kunststil Surrealismus",
+	"Artstyle Typography": "Kunststil Typografie",
+	"Artstyle Watercolor": "Kunststil Aquarell",
+	"Futuristic Biomechanical": "Futuristisch Biomechanisch",
+	"Futuristic Biomechanical Cyberpunk": "Futuristisch Biomechanischer Cyberpunk",
+	"Futuristic Cybernetic": "Futuristisch Kybernetisch",
+	"Futuristic Cybernetic Robot": "Futuristischer Kybernetischer Roboter",
+	"Futuristic Cyberpunk Cityscape": "Futuristische Cyberpunk-Stadtszene",
+	"Futuristic Futuristic": "Futuristisch",
+	"Futuristic Retro Cyberpunk": "Futuristischer Retro-Cyberpunk",
+	"Futuristic Retro Futurism": "Futuristischer Retro-Futurismus",
+	"Futuristic Sci Fi": "Futuristische Sci-Fi",
+	"Futuristic Vaporwave": "Futuristischer Vaporwave",
+	"Game Bubble Bobble": "Spiel Bubble Bobble",
+	"Game Cyberpunk Game": "Spiel Cyberpunk",
+	"Game Fighting Game": "Kampfspiel",
+	"Game Gta": "Spiel GTA",
+	"Game Mario": "Spiel Mario",
+	"Game Minecraft": "Spiel Minecraft",
+	"Game Pokemon": "Spiel Pokémon",
+	"Game Retro Arcade": "Retro-Arcade-Spiel",
+	"Game Retro Game": "Retro-Spiel",
+	"Game Rpg Fantasy Game": "RPG-Fantasy-Spiel",
+	"Game Strategy Game": "Strategiespiel",
+	"Game Streetfighter": "Spiel Streetfighter",
+	"Game Zelda": "Spiel Zelda",
+	"Misc Architectural": "Verschiedenes Architektonisch",
+	"Misc Disco": "Verschiedenes Disco",
+	"Misc Dreamscape": "Verschiedenes Traumlandschaft",
+	"Misc Dystopian": "Verschiedenes Dystopisch",
+	"Misc Fairy Tale": "Verschiedenes Märchen",
+	"Misc Gothic": "Verschiedenes Gotisch",
+	"Misc Grunge": "Verschiedenes Grunge",
+	"Misc Horror": "Verschiedenes Horror",
+	"Misc Kawaii": "Verschiedenes Kawaii",
+	"Misc Lovecraftian": "Verschiedenes Lovecraftian",
+	"Misc Macabre": "Verschiedenes Makaber",
+	"Misc Manga": "Verschiedenes Manga",
+	"Misc Metropolis": "Verschiedenes Metropole",
+	"Misc Minimalist": "Verschiedenes Minimalistisch",
+	"Misc Monochrome": "Verschiedenes Monochrom",
+	"Misc Nautical": "Verschiedenes Nautisch",
+	"Misc Space": "Verschiedenes Weltraum",
+	"Misc Stained Glass": "Verschiedenes Buntglas",
+	"Misc Techwear Fashion": "Verschiedenes Techwear-Mode",
+	"Misc Tribal": "Verschiedenes Stammes",
```

Comment:
Verschiedenes Tribalismus

Reply:
This suggestion has now been implemented